create.html

```diff
@@ -5,10 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/e98346ef52.js"
+      crossorigin="anonymous"
+    ></script>
     <title>Quiz App - Create</title>
   </head>
   <body>
-    <header><h1>Create</h1></header>
+    <header>
+      <h1>Create</h1>
+    </header>
```

Comment:
Just nit-picking: If you only have a `h1` element nested in your `header` you can work without a `header` element as the `h1` is semantic enough.